framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -320,6 +326,21 @@ public final Void visitClass(ClassTree classTree, Void p) {
         return null;
     }
 
+    /**
+     * If {@code tree} has a {@code @SuppressWarnings} add it to treesWithSuppressWarnings.
+     *
+     * @param tree a declaration on which a {@code @SuppressWarnings} annotation may be placed.
+     */
+    private void checkForSuppressWarningsAnno(Tree tree) {
```

Comment:
`checkForXXX` is rather confusing for a void function.
`storeXXX` maybe? Or `storeAnyXXX`?